Cargo.toml

```diff
@@ -43,6 +43,11 @@ edition = "2021"
 license = "MIT"
 repository = "https://github.com/denoland/deno"
 
+# Until https://github.com/ebkalderon/tower-lsp/pull/401 gets landed, we're
+# patching it in.
+[patch.crates-io]
+tower-lsp = { git = "https://github.com/denoland/tower-lsp.git", rev = "910e0f8" }
```

Comment:
Could you move it to the very bottom? When we're trying `deno_core` locally `[patch]` is usually placed at the bottom